.github/workflows/automation-ondemand-tests.yml

```diff
@@ -160,10 +160,12 @@ jobs:
             echo "image=${{ env.CHAINLINK_IMAGE }}" >>$GITHUB_OUTPUT
             echo "version=${{ github.sha }}" >>$GITHUB_OUTPUT
             echo "upgrade_version=${{ github.sha }}" >>$GITHUB_OUTPUT
```

Comment:
Should upgrade_version point to `env.UPGRADE_VERSION`?

Reply:
The logic here is a bit convoluted. If `inputs.chainlinkImage` is left empty, then we want to build new image from whichever commit this action is being run. So that's why `github.sha` is used. 
`env.UPGRADE_VERSION` is the version from which we want to upgrade from (only applies for upgrade test). So that would be the initial version and then the final version would be the build we specify, or the build that was just built.